pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -138,3 +150,23 @@ def get_fort_in_range(self):
             return fort
 
         return None
+
+    def get_fort_details(self, fort_id, latitude, longitude):
```

Comment:
This method that does caching probably shouldn't be here. We have more than one place that looks at fort information, so we should save that data and provide a function that everything can use. We should pull this out into __init__.py, or ideally a new file where we can put methods like this.